Core/StartupArgManager.cs

```diff
@@ -37,5 +39,33 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        private static void MigrateArgsFile()
+        {
+            const string ObsoleteFile = "laargs.txt";
+
+            try
+            {
+                if (File.Exists(ObsoleteFile) && string.IsNullOrWhiteSpace(File.ReadAllText(ObsoleteFile)))
+                {
+                    File.Delete(ObsoleteFile);
+                    ConsoleUtil.WriteLine("Obsolete configuration file 'laargs.txt' is empty and has been deleted.");
+                    return;
+                }
+
+                if (File.Exists(StartupArgsPath))
+                {
+                    ConsoleUtil.WriteLine("Unable to migrate your old 'laargs' configuration. The destination file already exists.");
```

Comment:
This will be displayed on EVERY start of LA when new config exists and old one either doesn't exist or is not empty. There is no need to display it (why inform user that old config can't be moved if the user already made a new config?) and it will be easiest the best way to remove this log.

Reply:
👍